uber/site_sections/jobs.py

```diff
@@ -170,10 +170,7 @@ def staffers_by_job(self, session, id, message=''):
         return {
             'job':       job,
             'message':   message,
-            'attendees': session.query(Attendee.id, Attendee.full_name)
-                                .filter_by(staffing=True, **({'trusted': True} if job.restricted else {}))
-                                .filter(Attendee.assigned_depts.contains(str(job.location)))
-                                .order_by(Attendee.full_name).all()
+            'attendees': session.staffers_by_job_options(job)
         }
```

Comment:
note: moved all this into its own function, ```session.staffers_by_job_options``` in order to be able to test it with new unit tests more easily.

this thing was previously one of my TODO's